scripts/pre_commit_linter.py

```diff
@@ -38,6 +38,15 @@
 4.  To lint files in verbose mode
         python -m scripts.pre_commit_linter --verbose
 
+5. To lint all file extensions or to lint just a specific file extension
```

Comment:
I think you don't need this: it's covered by point 6. I'm also a bit confused about how this demonstrates "to lint all file extensions".

Also, please be clear about the format of extension_1, extension_2, etc. The user won't know whether to type "*.py", ".py", "py", etc. I think you can just give the actual concrete example for how to do it with both JS and py, and the user should be able to generalize from there.

Reply:
Done.